www/kadm08/cv04/users.php

```diff
@@ -0,0 +1,13 @@
+<?php include __DIR__ . '/lib/usersDb.php'; ?>
+<?php include __DIR__ . '/includes/head.php'; ?>
+<?php include __DIR__ . '/includes/navigation.php'; ?>
+<main>
+    <h1>Users</h1>
+    <ul> 
+    <?php foreach(fetchUsers() as $user) {
+        echo '<li>' . $user['name'] . ', ' . $user['email'] . '</li>';

```

Comment:
html znacky nedavame do echo, pouze samotne hodnoty

Reply:
Opraveno, predelala jsem to do tabulky